ui/src/app/cohort-search/criteria-wizard/alerts/alerts.component.ts

```diff
@@ -0,0 +1,10 @@
+import {Component, Input} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+
+@Component({
+  selector: 'crit-alerts',
```

Comment:
shouldn't the selector be named 'app-alerts' to follow along with the other selectors and matching the component?